projects/build-basic/src/main/scala/sbt/tox4j/util/NativeFinder.scala

```diff
@@ -3,42 +3,40 @@
  */
 package sbt.tox4j.util
 
-import java.io.{ ByteArrayOutputStream, FileInputStream }
+import java.io.FileInputStream
 import java.lang.reflect.Method
 import javassist.util.proxy.{ MethodFilter, MethodHandler, ProxyFactory }
 
 import org.objectweb.asm._
 import sbt._
 
-// scalastyle:off
 object NativeFinder {
   def natives(classes: Set[File]): Map[String, Seq[String]] = {
-    var current: String = null
+    var currentClassName: String = null
```

Comment:
Limited the scope by putting it into the MethodHandler instance. Substituted Option for null.